Source/src/importers/ModelImporter.h

```diff
@@ -10,19 +10,23 @@ namespace Hachiko
     class ModelImporter final : public Importer
     {
     public:
-        ModelImporter();
+        ModelImporter() = default;
         ~ModelImporter() override = default;
         
         void Import(const char* path, YAML::Node& meta) override;
-        void ImportWithMeta(const char* path, YAML::Node& meta) override;
-        void Save(const Resource* resource) override;
-        Resource* Load(UID id) override;
-        void Delete(const YAML::Node& meta) override;
+        void Save(UID id, const Resource* resource) override
+        {
+            assert(false && "Importer will never save, since model is only an asset and not a resource");
+        };
+        // This should not be called, since model is not a resource
+        Resource* Load(UID id) override
+        {
+            assert(false && "Model importer will never load, since model is only an asset and not a resource");
+            return nullptr;
+        };
```

Comment:
Same with these, should we delete them?

Reply:
It would be cleaner but this way if anyone tries to call those he will be warned of his mistake

Reply:
Yeah, you're right, maybe we can just delete these from the base class and have them on needed sub-classes.